atomic_reactor/plugins/pre_change_from_in_df.py

```diff
@@ -99,15 +109,22 @@ def run(self):
         # update the parents in Dockerfile
         new_parents = []
         for parent in enclosed_parent_images:
+            if base_image_is_scratch(parent):
+                new_parents.append('scratch')
```

Comment:
Or `new_parents.append(parent)`? We avoid putting the string literal there in other places, I assume we'd want to avoid it here too.

Reply:
right